src/lib.rs

```diff
@@ -5,6 +5,8 @@
 #![cfg_attr(test, deny(warnings))]
 #![cfg_attr(all(test, feature = "nightly"), feature(test))]
 #![cfg_attr(docsrs, feature(doc_cfg))]
+#![cfg_attr(hyper_unstable_ffi, crate_type = "cdylib")]
+#![cfg_attr(hyper_unstable_ffi, allow(unknown_lints, deprecated_cfg_attr_crate_type_name))]
```

Comment:
```suggestion
#![cfg_attr(
    hyper_unstable_ffi,
    allow(unknown_lints, deprecated_cfg_attr_crate_type_name)
)]
```

Looks like rustfmt wants this instead.

Reply:
Ah sorry, I didn't notice that, it's fixed now.